### PR TITLE
feat(angularjs): daterangepicker component

### PR DESCRIPTION
**Pull request by bndynet**
## Description

This is a new angularjs component for date range picker.

## Motivation and Context

Currently, Momentum UI does not have the angularjs component to pick a date range.

## Screenshots:

**Default**
image

**With Date Range Options**
image

**Align Right**
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by bndynet** [comment]
@pauljeter and @ArthusLiang Could you help to review this component? I think I did not change any visualization but got above **percy/momentum-ui 46 visual changes** issue. I have no idea what's wrong.

File: icons/svg/call-voicemail_18.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ArthusLiang) [inline icons/svg/call-voicemail_18.svg]: Are theses icons related to this update?

Comment (bndynet) [reply icons/svg/call-voicemail_18.svg]: No, my branch is just for date range picker component. I did not change or add any icons. That may be from master branch.

Comment (pauljeter) [reply icons/svg/call-voicemail_18.svg]: Can you `git pull --rebase` on your branch against the master branch? That should clear up these extra commits.

Comment (bndynet) [reply icons/svg/call-voicemail_18.svg]: Ok, got it.

File: core/scss/components/date-range-picker/date-range-picker.scss [hunk 2]
```diff
@@ -45,4 +45,111 @@
...
+  // for angularjs
+  md-date-range-picker.angularjs {
+    display: inline-block;
+
+    .md-date-range-picker {
+      position: relative;
+      display: inline-block;
+      width: 100%;
+
+      &.with-icon {
```
Comment (bfbiggs) [inline core/scss/components/date-range-picker/date-range-picker.scss]: I suggest either moving this into NG or adjusting these class names for bems.  As this date range picker goes beyond what our current scss covers @pauljeter

File: angularjs/src/lib/components/daterangepicker/daterangepicker.component.ts [hunk 3]
```diff
@@ -0,0 +1,264 @@
+/** @component date-range-picker */
+import * as _ from 'lodash';
```
[the remark this answers is not in the document]
Comment (bndynet) [reply angularjs/src/lib/components/daterangepicker/daterangepicker.component.ts]: Thanks for your suggestions.

**Pull request closed without merging**